CMakeLists.txt

```diff
@@ -313,10 +313,10 @@ set(OFFSCREEN_SOURCES
   src/OpenCSGRenderer.cc)
 
 set(GUI_SOURCES
-  src/MainWindow.cc
+  src/MainWindow.h
   src/OpenSCADApp.cc
   src/WindowManager.cc
-  src/AboutDialog.cc
+  src/AboutDialog.h
```

Comment:
We shouldn't add headers here, but create a separate cmake variable, e.g. `GUI_HEADERS` and add it there.
Not adding headers separately was an oversight on my part, as I forgot that we have some header-only code.